### DISCUSSION TITLE
Harvesting Maize for silage

### DISCUSSION THREAD

**Discussion by IrisVogeler** [APSIMInitiative/ApsimX#9233] [Q&A] [answered]
I am trying to harvest maize for silage at a certain phonology stage.  I have tired the following, but this does not work.  Any better idea?
if (Crop.Phenology.BBCH.Stage) >80 then

**Comment by BrianCollinss** [comment, marked as answer]
You can change IPlant to Plant to get access to Crop.Phenology:

![image](https://github.com/user-attachments/assets/e6299137-fd59-4131-8e65-2d15bfbfe446)

Or, 

![image](https://github.com/user-attachments/assets/a2bb9541-32ac-42f3-9942-f909dc6964c7)

Or, https://github.com/APSIMInitiative/ApsimX/discussions/9133#discussioncomment-10090941

Also, there is a manager script in 'Management Toolbox' that might give you some ideas. I am sure there are simpler ways as well.

![image](https://github.com/user-attachments/assets/e5ac880b-aca3-4e2c-ad04-8215f3d2d83a)

**Comment by IrisVogeler** [comment]
Thanks Brian, got it working

**Comment by BrianCollinss** [comment]
No worries. If your question has been answered, you can go ahead and mark my response as the answer and close the thread.

**Reply by BrianCollinss** [reply]
@IrisVogeler you should mark the first response as the acceptable response :)
